docs/_data/navigation.yml

```diff
@@ -4,7 +4,7 @@ main:
     target: target="_blank" rel="noopener noreferrer"
     section: /docs/
   - title: Tutorial
-    url: /docs/get_started/kubernetes_tutorial.html
+    url: /docs/tutorial-tiles/kubernetes/kubernetes-tutorial-base.html
```

Comment:
- [ ] can we just call this directory `tutorials/`?
- [ ] can docs/get_started/kubernetes_tutorial.html redirect to the new tutorial start page?
- [ ] can you log an issue in the docs repo to update the tutorial links there and note `Dependent on cyberark/secretless-broker#666`

Reply:
Updated and created issue but regarding pt 3, should we update links in docs if we have a redirect in place?